src/frontend/apps/e2e/__tests__/app-desk/teams-create.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect, test } from '@playwright/test';
+
+import { keyCloakSignIn } from './common';
+
+test.beforeEach(async ({ page, browserName }) => {
+  await page.goto('/');
+  await keyCloakSignIn(page, browserName);
+});
+
+test.describe('Teams', () => {
+  test('checks all the create box elements are visible', async ({ page }) => {
```

Comment:
Your test checks the layout/ elements visibility + the cancel button

Reply:
[30e0c2f](https://github.com/numerique-gouv/people/pull/43/commits/30e0c2fb85973c4fa090ec739429b1535193d1a2)